DatadogCore/Sources/Datadog.swift

```diff
@@ -492,6 +492,8 @@ public enum Datadog {
             verbosityLevel: { Datadog.verbosityLevel }
         )
 
+        registerObjcExceptionHandlerOnce()
```

Comment:
suggestion/ It might make sense to register this handler before we instantiate `DatadogCore`. As part of core init SDK interacts with file system (to create directory structure), which I believe takes advantage of obj-c error handling.

Reply:
Yes, good point 👍